server/src/main/java/org/candlepin/resource/OwnerResource.java

```diff
@@ -1054,50 +1054,14 @@ public void importManifest(
         @PathParam("owner_key") @Verify(Owner.class) String ownerKey,
         @QueryParam("force") String[] overrideConflicts, MultipartInput input) {
 
-        if (overrideConflicts.length == 1) {
-            /*
-             * For backward compatibility, look for force=true and if found,
-             * treat it just like what it used to mean, ignore an old manifest
-             * creation date.
-             */
-            if (overrideConflicts[0].equalsIgnoreCase("true")) {
-                overrideConflicts = new String [] { "MANIFEST_OLD" };
-            }
-            else if (overrideConflicts[0].equalsIgnoreCase("false")) {
-                overrideConflicts = new String [] {};
-            }
-        }
-        if (log.isDebugEnabled()) {
-            for (String s : overrideConflicts) {
-                log.debug("Forcing conflict if encountered: " + s);
-            }
-        }
-
-        ConflictOverrides overrides = null;
-        try {
-            overrides = new ConflictOverrides(overrideConflicts);
-        }
-        catch (IllegalArgumentException e) {
-            throw new BadRequestException(i18n.tr("Unknown conflict to force"));
-        }
-
-        String filename = "";
+        ConflictOverrides overrides = createConflictOverrides(overrideConflicts);
         Owner owner = findOwner(ownerKey);
         Map<String, Object> data = new HashMap<String, Object>();
+        String filename = "";
         try {
             InputPart part = input.getParts().get(0);
-            MultivaluedMap<String, String> headers = part.getHeaders();
-            String contDis = headers.getFirst("Content-Disposition");
-            StringTokenizer st = new StringTokenizer(contDis, ";");
-            while (st.hasMoreTokens()) {
-                String entry = st.nextToken().trim();
-                if (entry.startsWith("filename")) {
-                    filename = entry.substring(entry.indexOf("=") + 2, entry.length() - 1);
-                    break;
-                }
-            }
-            File archive = part.getBody(new GenericType<File>() {
-            });
+            filename = extractFileNameFromHeaders(part);
+            File archive = part.getBody(new GenericType<File>() {});
             log.info("Importing archive " + archive.getAbsolutePath() +
```

Comment:
Since you're doing a pretty significant refactor of this area, can you also fix the log lines to use substitution instead of concatenation?

```
log.info("Importing archive: {}", archive.getAbsolutePath());
```